jdi-light/src/main/java/com/epam/jdi/light/settings/WebSettings.java

```diff
@@ -94,6 +95,7 @@ public static void inView() {
 
     public static ElementArea CLICK_TYPE = SMART_CLICK;
     public static TextTypes TEXT_TYPE = SMART_TEXT;
+    public static SetTextTypes SET_TEXT_TYPE = SET_TEXT;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4522228)